demo/build.gradle

```
@@ -39,15 +39,16 @@ android {
     }
 }
 
+// [START android_maps_utils_install_snippet]
```

```suggestion
// [START maps_android_utils_install_snippet]
```

region tags need to start with `maps_`